### PR TITLE
Fix AMD GPU memory reading

### PR DESCRIPTION
**Pull request by omichel**
As reported by a user the GPU memory reading on AMD graphics cards was broken, probably due to an update of the AMD driver. This PR fixes the problem.

### CODE REVIEW

**Comment by omichel** [comment]
test me

**Comment by omichel** [comment]
👍

**Comment by omichel** [comment]
test me

**Comment by omichel** [comment]
👍

File: src/wren/GlState.cpp [hunk 1]
```diff
@@ -120,15 +120,18 @@ namespace wren {
...
-              << std::endl);
-
+      else {
+        // Try to use GL_TEXTURE_FREE_MEMORY_ATI:
+        // it seems to be working even if the corresponding GLAD_GL_ATI_meminfo is not available
+        GLint array[4];
+        array[0] = -1;
+        glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, array);
+        cGpuMemory = array[0];
+        while (glGetError()) {  // skip any possible GL_INVALID_ENUM error
```
Comment (DavidMansolino) [inline src/wren/GlState.cpp]:
```suggestion
        while (glGetError() != GL_NO_ERROR) {  // skip any possible GL_INVALID_ENUM error
```

File: src/wren/GlState.cpp [hunk 2]
```diff
@@ -120,15 +120,18 @@ namespace wren {
...
-        DEBUG("GL_TEXTURE_FREE_MEMORY_ATI and GL_GPU_MEMORY_INFO_TOTAL_AVAILABLE_MEMORY_NVX extensions not supported by "
-              "hardware, impossible to detect GPU memory"
-              << std::endl);
-
+      else {
+        // Try to use GL_TEXTURE_FREE_MEMORY_ATI:
+        // it seems to be working even if the corresponding GLAD_GL_ATI_meminfo is not available
+        GLint array[4];
+        array[0] = -1;
+        glGetIntegerv(GL_TEXTURE_FREE_MEMORY_ATI, array);
```
Comment (DavidMansolino) [inline src/wren/GlState.cpp]: We should probably call: https://github.com/omichel/webots/blob/c99c027abbb4316d52abe46980398ca4c568772a/src/wren/GlState.cpp#L732
Just before this to make sure that we don't skip other error while skipping the error generated by this enum.

**Comment by omichel** [comment]
test me

**Comment by omichel** [comment]
test me

**Pull request merged**